include/networkit/auxiliary/PrioQueue.hpp

```diff
@@ -198,6 +207,16 @@ inline uint64_t Aux::PrioQueue<Key, Value>::size() const {
     return pqset.size();
 }
 
+template<class Key, class Value>
+inline bool Aux::PrioQueue<Key, Value>::empty() const noexcept {
```

Comment:
`inline` is redundant for templates. Template functions are always inline.

Reply:
I removed `inline` in the other methods as well.